### PR TITLE
Scotland Class 5 - Abdulrahman Salman - Zoo webpage - Week 1

### PR DESCRIPTION
**Pull request by Abadi11**

### CODE REVIEW

File: zoo-css-challenge/index.html [hunk 1]
```diff
@@ -78,6 +79,19 @@ <h3>Height and Size</h3>
 			<p>Giraffes use their height to good advantage and browse on leaves and buds in treetops that few other animals can reach (acacias are a favorite). Even the giraffe's tongue is long! The 21-inch tongue helps them pluck tasty morsels from branches. Giraffes eat most of the time and, like cows, regurgitate food and chew it as cud. A giraffe eats hundreds of pounds of leaves each week and must travel miles to find enough food.</p>
 		</section>
 
+		<section class="horse">
+			<h3>Horses</h3>
```
Comment (bonboh) [inline zoo-css-challenge/index.html:83]:
```suggestion
			<h2>Horses</h2>
```
All the other animal sections have `h2` as their first child element so it makes sense to have this as `h2` as well so it's at the same heading level

File: zoo-css-challenge/index.html [hunk 2]
```diff
@@ -78,6 +79,19 @@ <h3>Height and Size</h3>
...
+			<p> <img class="image-horse" src="./horses.jpg" alt="Horses">Horses and humans have an ancient relationship. Asian nomads probably domesticated the first horses some 4,000 years ago, and the animals remained essential to many human societies until the advent of the engine. Horses still hold a place of honor in many cultures, often linked to heroic exploits in war.
+				</p>
+				<br>
+				<br>
+				<br>
+				<br>
```
Comment (bonboh) [inline zoo-css-challenge/index.html:89]:
```suggestion
```
`<br>` is not really used by frontend developers. Instead, we would use CSS to add margins to create space below the `p` here

File: zoo-css-challenge/index.html [hunk 3]
```diff
@@ -90,6 +104,20 @@ <h4>Helping wild turtles in Seychelles</h4>
...
+			<div class="card">
+				<time>
+					<p id="date"></p>
+					<script>
+						n =  new Date();
+						y = n.getFullYear();
+						m = n.getMonth() + 1;
+						d = n.getDate();
+						document.getElementById("date").innerHTML = y + "-" + m + "-" + d;
+					</script>
```
Comment (bonboh) [inline zoo-css-challenge/index.html:116]:
```suggestion
					2021-06-04
```
This task was okay with using plain text like the other cards. Also, it's not common to have `<script>` in the middle of HTML, and we haven't done JS yet in the course so no need to use JS here 😄

File: zoo-css-challenge/style.css [hunk 4]
```diff
@@ -30,7 +32,7 @@ section {
 }
 
 img {
-  border: 5px solid #8075ff;
+  border: none;
```
Comment (bonboh) [inline zoo-css-challenge/style.css:35]:
```suggestion
```
We could also delete the whole `img` selector, as an `img` doesn't have a border by default

File: zoo-css-challenge/style.css [hunk 5]
```diff
@@ -160,19 +199,34 @@ footer {
...
+
+}
+
+.education-container button:hover{
+  background-color: blueviolet;
+} 
+
+.education-container button:focus{
+  background-color: blueviolet;
+}
```
Comment (bonboh) [inline zoo-css-challenge/style.css:228]:
```suggestion
.education-container button:hover,
.education-container button:focus{
  background-color: blueviolet;
} 
```
We can combine the two selectors as they both have the same CSS declaration

**Review by bonboh — APPROVED** [review]
Great job @Abadi11! I left some small suggestions for you to read, but otherwise you don't need to update this PR as it's good 👍

File: zoo-css-challenge/index.html (hunk 1 above)
Comment (Abadi11) [reply zoo-css-challenge/index.html:83]: Great, Thank you very much

File: zoo-css-challenge/index.html (hunk 2 above)
Comment (Abadi11) [reply zoo-css-challenge/index.html:89]: Great, Thank you very much

File: zoo-css-challenge/index.html (hunk 3 above)
Comment (Abadi11) [reply zoo-css-challenge/index.html:116]: 😅, I did not know how to do without it 

Thank you very much

File: zoo-css-challenge/style.css (hunk 4 above)
Comment (Abadi11) [reply zoo-css-challenge/style.css:35]: Great, Thank you very much

File: zoo-css-challenge/style.css (hunk 5 above)
Comment (Abadi11) [reply zoo-css-challenge/style.css:228]: Ok, we can by comma, Thank you very much

**Pull request closed without merging**